tasks/request_status_for_granule/API.md

```diff
@@ -164,7 +159,7 @@ Entry point for the request_status_for_granule Lambda.
 - `'restore_destination'` _str_ - The name of the glacier bucket
   the file is being copied to.
 - `'status'` _str_ - The status of the restoration of the file.
-  May be 'pending', 'staged', 'success', or 'failed'.
+  May be 'pending', 'staged', 'success', or 'error'.
```

Comment:
Fixed this in a lot of places. DB table uses the word "error", and that's what we key off of.